Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9.0-buster
+
+RUN sed -i 's/deb.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
+RUN sed -i 's/security.debian.org/mirrors.aliyun.com/g' /etc/apt/sources.list
+
+RUN pip install --upgrade pip && pip install mkdocs mkdocs-material
+
+WORKDIR /app
+
+COPY codes /app/codes
```

Comment:
Maybe `Add . /app` is clearer.

Reply:
If use `COPY . /app`, other files like `LICENSE` or `.gitignore` will be copied into the **DOCKER**, which is unnecessary.